README.md

```diff
@@ -1,6 +1,15 @@
 # census-rm-load-generator
 Performance testing simulated load generator
 
+This app loads a configured (Config.CASES_TO_FETCH) number of cases from the database and a configured 
+(Config.UNADDRESSED_QIDS_TO_FETCH) number of unlinked qids from the database.
```

Comment:
I think it's worth saying explicitly that a big enough sample load and unaddressed batch have to be run into the environment before running this.